internal/rpc/jsonrpc/methods.go

```diff
@@ -133,7 +133,7 @@ var handlers = map[string]handler{
 	"walletlock":              {fn: (*Server).walletLock},
 	"walletpassphrase":        {fn: (*Server).walletPassphrase},
 	"walletpassphrasechange":  {fn: (*Server).walletPassphraseChange},
-
+	"walletpubpassphrasechange": {fn: (*Server).walletPubPassphraseChange},
```

Comment:
There was an empty line space here that demarcated 2 sections of code. Restore the empty line space after the `walletpubpassphrasechange` line.

Reply:
Restored the empty line space demarcating sections of code